docs/guides/networking/ssh/advanced-ssh-tunneling/index.md

```diff
@@ -0,0 +1,216 @@
+Chrome and Chromium-based Browsers: https://chrome.google.com/webstore/detail/foxyproxy-standard/gcknhkkoolaabfmlnjonogaaifnjlfnp?hl=en
+Firefox: https://addons.mozilla.org/en-US/firefox/addon/foxyproxy-standard/
+```
+
+Once enabled, click on the FoxyProxy icon in your toolbar and go to "**Options**". 
+
+- To add your proxychains set-up to FoxyProxy, click on "**Add**".
+
+- On the next screen, set "**Proxy Type**" to `SOCKS5`.  
+
+- Set "**Proxy IP address**" to `127.0.0.1`, "**localhost**" or your local machine's IP address.
+
+- Set "**Port**" to `9050`.
+
+- You are free to set your proxy's name and color to whatever you prefer.
+
+- Close the FoxyProxy tab in your browser.
+
+- Click on the FoxyProxy icon in your toolbar. You should now see your recently added proxy.  Click on your added proxy to run your browser through proxychains.
+
+- You should be able to browse to previously inaccessible web servers on the network you connected to.
+
+{{< note >}}
+
+The above options assume you are using the default proxychains settings, though if you changed your proxychains configuration file to use a different protocol or port, reflect your custom options when adding your proxy to FoxyProxy.
+
+{{< /note >}}
+
+
+
+## Remote Port Forwarding
+
+Remote port forwarding is a similar concept to local port forwarding, however, instead of forwarding a local port and local service to a remote network (ssh server's network), we are instead pulling a remote network's resource into our local machine. In other words, the SSH server will listen for any connection to a port we specify and then tunnel any traffic to our local SSH client and forwards it to our destination port.
+
+```bash
+ssh -R [REMOTE:]REMOTE_PORT:DESTINATION:DESTINATION_PORT [USER@]SSH_SERVER
+```
+
+Using our example IPs, we would run the following in our terminal.
+
+```bash
+ssh -R 8080:127.0.0.1:3000 user@10,10,10,2
+```
+
+Remote port forwarding is often used to provide access to an internal service to a client outside of that network.  The example above would allow you to access a web server running on `10.10.10.100:8080` by sending your browser to `http://127.0.0.1:3000`. 
+
+
+
+## Keeping Sessions Alive
+
+It is quite likely that a situation may occur where your connection is momentarily dropped or your IP address changes due to external factors. Unfortunately, you will need to manually re-establish your connection as there is no native solution in your SSH client, though a simple bash script does alleviate this problem.  If your connection happens to drop, it will automatically and continuously attempt to reconnect.  Remember to change the values for the `IP` (127.0.0.1), `user`, SSH server (`host`), and `ports` (8080) before you run the following in your terminal.
+
+```bash
+while true; do
+    ssh -N -oExitOnForwardFailure=yes -L 8080:127.0.0.1:8080 user@host
+    sleep 1
+done
+```
+
+## Multi-Hop
+
+Chain multiple SSH tunnels together using one command with the `-t` flag.

```

Comment:
Is there a use case or scenario in which this is favored?

Reply:
added.